tests/sles4sap/saptune/mr_test.pm

```diff
@@ -75,6 +75,27 @@ sub tune_baseline {
     assert_script_run "sed -ri -e '/fs\\/file-max/s/:([0-9]*)\$/:~~\\1/' -e '/:scripts\\/shm_size/s/:([0-9]*)\$/:~~\\1/' $filename";
 }
 
+sub test_bsc1152598 {
+    my ($self) = @_;
+
+    my $SLE = is_sle(">=15") ? "SLE15" : "SLE12";
```

Comment:
Please use `' [...] '` instead of `" [...] "` when there is no variable in the text (like you did below). Some qa-sle reviewer prefers this (me too ;-)).

Reply:
It's a matter of personal taste, but in the future when I refactor the code I willl change all strings.